pepsico/monthly_seasonal_conv_mapping.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+import plotly.express as px
+from dash import Dash, dcc, html, Input, Output
+from urllib.request import urlopen
+import json
```

Comment:
I don't think any of those are used? are they? if not, remove them. I am not sure numpy is used either.

Reply:
ok, I removed them